src/components/InputWithIcon.jsx

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Select from 'react-select';
+import './InputWithIcon.less';
+
+class InputWithIcon extends React.Component {
+  render() {
+    return (
+      <React.Fragment>
+        <div className={'input-with-icon'.concat(this.props.className ? ` ${this.props.className}` : '')}>
+          {
+            this.props.inputOptions ? (
+              <Select
+                className={this.props.inputClassName}
+                value={this.props.inputValue}
+                placeholder={this.props.inputPlaceholderText}
+                options={this.props.inputOptions}
+                onChange={this.props.inputOnChange}
+                multi={this.props.isMulti}
+              />
+            ) : (
+              <input
+                type='text'
+                className={this.props.inputClassName}
+                onBlur={this.props.inputOnChange}
+              />
+            )
+          }
+          {
+            this.props.shouldDisplayIcon ? (
+              <this.props.iconSvg
+                className={'input-with-icon__icon'.concat(this.props.iconClassName ? ` ${this.props.iconClassName}` : '')}
+              />
+            ) : null
+          }
+        </div>
+        {
+          this.props.shouldDisplayText ?
+            <p
+              className={'input-with-icon__text'.concat(this.props.textClassName ? ` ${this.props.textClassName}` : '')}
```

Comment:
there is a space before `this.props.textClassName` so I think that works right? I wanted to make this component reusable, not just for the use case I'm using it for now, so I added the option for this class. In an older version of the spec, there was text to be displayed under the input section if the submitter ID was invalid, so I left this in in case 1) we go back to that version of the spec or 2) this use case presents itself in the future.

similarly to your comment above about just using the checkmark icon, we could do that but I think that this component is more reusable if we allow the icon to be passed as a prop.